.github/workflows/helm.yml

```diff
@@ -45,8 +48,8 @@ jobs:
       run: |
         export SHELL=/bin/bash
         eval $(minikube -p minikube docker-env)
-        docker pull cvat/server
-        docker pull cvat/ui
+        docker pull cvat/server:dev
+        docker pull cvat/ui:dev
```

Comment:
Maybe it's better to build these images locally, because in case when some PR change both server code and helm templates this workflow won't catch potential regression, since this workflow works only with published dev version of images